.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Bug
+about: "Report confirmed bugs. For unconfirmed bugs please visit https://discuss.elastic.co/c/beats"
+
+---
+
+Please post all questions and issues on https://discuss.elastic.co/c/beats
```

Comment:
This should be targeted to the agent discuss most likely https://discuss.elastic.co/c/elastic-stack/elastic-agent